test/onnx/test_operators.py

```diff
@@ -877,6 +877,20 @@ def test_det(self):
         x = torch.randn(2, 3, 5, 5, device=torch.device('cpu'))
         self.assertONNX(lambda x: torch.det(x), x, opset_version=11)
 
+    def test_softmaxcrossentropy(self):
```

Comment:
can we have one 3d-mean, one 3d-none and one 4d test? 
they all pass by different branches in the postpass

Reply:
Should we also have tests that have inputs bigger than 4d, like 5d or 6d?